### DISCUSSION TITLE
"Bad" Connections on all mirrors

### DISCUSSION THREAD

**Discussion by Camadams1179** [termux/termux-packages#18989] [Q&A]
![Screenshot_20240114-225504_Termux](https://github.com/termux/termux-packages/assets/109619090/0f9af8b9-1e0d-44ff-9c02-80b461c92e5f)

Per the image I've provided, I've yet to find a solution, this is an issue among all terminals I've used.

Im using termux off F-droid, I'm running the latest crdroid version (10.1) on OnePlus 9 5G

Things I've already done:
-Tried all available mirrors
-Edited repo list to include root and x11 packages
-Uninstall and install termux
-turn off all network settings and back on
-turned off AdAway/VPN/DNS
-pinged various sites (e.g Google.com brings "unknown host")
-switched to non 5G wifi
-verified termux is allowed Internet access

I'm unsure if this is my newer phone version, incompatibility of my device (despite members in crdroid are able to use termux fully), or just a user error (which is the most likely issue). 

Any suggestions are welcomed and I'll provide any clarification needed.

**Comment by agnostic-apollo** [comment]
Possible duplicate of https://github.com/termux/termux-app/issues/3761

**Reply by agnostic-apollo** [reply]
If you do have root access, then run `su -c 'auditctl -r 0'` and then `apt update` and then `su -c 'logcat -d > /sdcard/logcat.txt` and check `avc` denial messages or post `logcat.txt` here after removing any private info or email me at agnosticapollo@gmail.com. Do this now before uninstalling.

**Reply by Camadams1179** [reply]
There's no output in my directory? Termux doesn't acknowledge the commands either

**Reply by agnostic-apollo** [reply]
huh? What directory? What does "doesn't acknowledge" mean?

Check edit in above comment, need to run `logcat -d`.

**Reply by Camadams1179** [reply]
> huh? What directory? What does "doesn't acknowledge" mean?

Like it doesn't do anything or spit out an error 

> Check edit in above comment, need to run `logcat -d`.

Got it will update soon 

Edit: I sent an email with logcat
